packages/ember-extension-support/tests/container_debug_adapter_test.js

```diff
@@ -1,52 +1,47 @@
+import { moduleFor, ApplicationTestCase } from 'internal-test-helpers';
+import { assign } from 'ember-utils';
 import { run } from 'ember-metal';
 import { Controller as EmberController } from 'ember-runtime';
 import '../index'; // Must be required to export Ember.ContainerDebugAdapter.
-import { Application } from 'ember-application';
 
-let adapter, App, appInstance;
+let adapter;
 
-function boot() {
-  run(App, 'advanceReadiness');
-}
+moduleFor('Container Debug Adapter', class extends ApplicationTestCase {
+  constructor() {
+    super();
+    adapter = this.application.__deprecatedInstance__.lookup('container-debug-adapter:main');
```

Comment:
Using the `autoboot: true` did not create an application instance, so I had to keep `__deprecatedInstance__ `.